website/src/hugo/content/changelog.md

```diff
@@ -8,122 +8,168 @@
 Maintenance branches are merged before each new release. This change log is
 ordered chronologically, so each release contains all changes described below it.
 
+# v0.23.7 (2021-12-07)
```

Comment:
It's fine to have this into `series/0.22`, I suppose. Then we don't have to create another backport-PR. Those changes will flow into `series/0.23` naturally when merging.